Marlin/LED_Strip.cpp

```diff
@@ -0,0 +1,32 @@
+// RGB Led Strip Control
+ 
+#include "Marlin.h"
+#include "temperature.h" 
+#include "Configuration.h"
+#include "ultralcd.h"
+#include "LED_Strip.h"
+
+#if ENABLED(LED_STRIP)
+
+// Set color by RGB values
+void rgb_to_led_strip(int red, int green, int blue) {    
+  analogWrite(REDPIN, red);
+  analogWrite(GREENPIN, green);
+  analogWrite(BLUEPIN, blue);
```

Comment:
If this needs to require PWM-able pins, we actually have a way to check for that, which works for any board.